enrollment-server-onboarding/src/test/java/com/wultra/app/onboardingserver/statemachine/DocumentVerificationTransitionsTest.java

```diff
@@ -143,14 +140,7 @@ private void testDocumentVerificationStatus(IdentityVerificationStatus identityS
     }
 
     private IdentityVerificationEntity createIdentityVerificationLocal(IdentityVerificationPhase phase, IdentityVerificationStatus status) {
-        final DocumentVerificationEntity documentVerification = new DocumentVerificationEntity();
```

Comment:
This was introduced in #501 but not needed anymore, because now we end at `DOCUMENT_VERIFICATION_FINAL_IN_PROGRESS` (no event is triggered internally).